### PR TITLE
Down state token updates 

### PR DESCRIPTION
**Pull request by nabuhasan**
## Description

- Replaced **component-size-difference-down** token to a dimension token with a value of -2px. To do this, I created a new dimension token and deleted the old token under sizing category in TS. I also referenced the same UUID in the new token since they are the same just reordered to a difference section. It didn't seem like there was a way for me to drag the token to a different category. 

- Updated **component-size-width-ratio-down** value from 3px to 0.3333 number token. 

These updates were made to both desktop and mobile. 

## Motivation and context
Inaccuracies in token values of a [previous branch](https://github.com/adobe/spectrum-tokens-studio-data/pull/27) needed to be addressed. 

## Related issue

Added 3 non-color tokens for S2 component down states https://github.com/adobe/spectrum-tokens-studio-data/pull/27

## Types of changes

- [ ] Patch (bug fixes, typos, mistakes; non-breaking change which fixes an issue)
- [X] Minor (add a new token, changing a value; non-breaking change which adds functionality)
- [ ] Major (deleting a token, changing token value type; fix or feature that would cause existing functionality to change)

## Checklist:

- [X] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [X] I updated the token in all applicable sets. This applies if updating, adding, or deleting a token that has data across different sets (for example, if the value differs across color themes.)

### CODE REVIEW

**Comment by mrcjhicks** [comment]
**[TESTING spectrum2-non-colors](https://github.com/adobe/spectrum-tokens-studio-data/actions/runs/7481374033)**

READING spectrum-tokens: `/spectrum-tokens/packages/tokens/src/`
READING   tokens studio: `/spectrum-tokens-studio-data/src/tokens-studio/spectrum2-non-colors/`
TOKENS STUDIO DATA IS NON-COLOR ONLY

<details><summary>SPECTRUM SYNC SUMMARY</summary><p>

```
NEW TOKENS: (6) component-size-width-ratio-down/desktop, component-size-minimum-perspective-down/desktop, component-size-difference-down/desktop, component-size-width-ratio-down/mobile, component-size-minimum-perspective-down/mobile, component-size-difference-down/mobile
CHANGED VALUES: (26) workflow-icon-size-100/desktop, workflow-icon-size-200/desktop, workflow-icon-size-300/desktop, workflow-icon-size-100/mobile, workflow-icon-size-200/mobile, workflow-icon-size-300/mobile, workflow-icon-size-50/mobile, workflow-icon-size-75/mobile, component-top-to-workflow-icon-100/desktop, component-top-to-workflow-icon-200/desktop, component-top-to-workflow-icon-300/desktop, text-to-visual-100/desktop, text-to-visual-200/desktop, text-to-visual-300/desktop, text-to-visual-50/desktop, text-to-visual-75/desktop, component-top-to-workflow-icon-100/mobile, component-top-to-workflow-icon-200/mobile, component-top-to-workflow-icon-300/mobile, component-top-to-workflow-icon-50/mobile, component-top-to-workflow-icon-75/mobile, text-to-visual-100/mobile, text-to-visual-200/mobile, text-to-visual-300/mobile, text-to-visual-50/mobile, text-to-visual-75/mobile impacting 15 tokens
```
</p></details>

TEST COMPLETE

**Comment by GarthDB** [comment]
Merging this because it is just a change from a previously merged pr that has already been approved and merged.

**Comment by mrcjhicks** [comment]
**[SYNCING spectrum2-non-colors](https://github.com/adobe/spectrum-tokens-studio-data/actions/runs/7481793571)**

READING spectrum-tokens: `/spectrum-tokens/packages/tokens/src/`
READING   tokens studio: `/spectrum-tokens-studio-data/src/tokens-studio/spectrum2-non-colors/`
TOKENS STUDIO DATA IS NON-COLOR ONLY

<details><summary>SPECTRUM SYNC SUMMARY</summary><p>

```
NEW TOKENS: (6) component-size-width-ratio-down/desktop, component-size-minimum-perspective-down/desktop, component-size-difference-down/desktop, component-size-width-ratio-down/mobile, component-size-minimum-perspective-down/mobile, component-size-difference-down/mobile
CHANGED VALUES: (26) workflow-icon-size-100/desktop, workflow-icon-size-200/desktop, workflow-icon-size-300/desktop, workflow-icon-size-100/mobile, workflow-icon-size-200/mobile, workflow-icon-size-300/mobile, workflow-icon-size-50/mobile, workflow-icon-size-75/mobile, component-top-to-workflow-icon-100/desktop, component-top-to-workflow-icon-200/desktop, component-top-to-workflow-icon-300/desktop, text-to-visual-100/desktop, text-to-visual-200/desktop, text-to-visual-300/desktop, text-to-visual-50/desktop, text-to-visual-75/desktop, component-top-to-workflow-icon-100/mobile, component-top-to-workflow-icon-200/mobile, component-top-to-workflow-icon-300/mobile, component-top-to-workflow-icon-50/mobile, component-top-to-workflow-icon-75/mobile, text-to-visual-100/mobile, text-to-visual-200/mobile, text-to-visual-300/mobile, text-to-visual-50/mobile, text-to-visual-75/mobile impacting 15 tokens
```
</p></details>

WROTE: `/spectrum-tokens/packages/tokens/src/layout.json`

SYNC COMPLETE

[Spectrum Tokens Pull Request](https://github.com/adobe/spectrum-tokens/pull/240)

File: src/tokens-studio/spectrum2-non-colors/spectrum2/layout/desktop.json [hunk 1]
```diff
@@ -395,19 +395,8 @@
...
-        "uuid": "9ef4a0a5-b81c-4f54-8927-286ece29a824",
-        "name": "component-size-difference-down",
-        "constant-token-duplicate": false
-      }
-    }
-  },
   "component-size-width-ratio-down": {
-    "value": "3px",
+    "value": "0.3333",
     "type": "sizing",
```
Comment (karstens) [inline src/tokens-studio/spectrum2-non-colors/spectrum2/layout/desktop.json:400]: Hi @nabuhasan @lynnhao I totally missed that this token might not be of type 'sizing'. I think, this 'ratio' token and other tokens (like `text-field-minimum-width-multiplier`) could be of type 'number'. I will scan through all the non-color tokens tomorrow and create a list of tokens, that might not be sizing tokens, if that is alright with you. Thanks Frank

Comment (nabuhasan) [reply src/tokens-studio/spectrum2-non-colors/spectrum2/layout/desktop.json:400]: No problem, let me know where specific tokens should live and I can make the updates then! @karstens

**Pull request merged**